pandas/tests/io/test_html.py

```diff
@@ -641,6 +669,224 @@ def test_different_number_of_rows(self):
         res = self.read_html(out, index_col=0)[0]
         tm.assert_frame_equal(expected, res)
 
+    def test_colspan_rowspan_are_1(self):
+        # GH17054
+        expected = """<table>
+                        <tr>
+                          <th>X</th>
+                          <th>Y</th>
+                          <th>Z</th>
+                          <th>W</th>
+                        </tr>
+                      </table>"""
+        out = """<table>
+                   <tr>
+                     <th colspan="1">X</th>
+                     <th>Y</th>
+                     <th rowspan="1">Z</th>
+                     <th>W</th>
+                   </tr>
+                 </table>"""
+        expected = self.read_html(expected)[0]
+        res = self.read_html(out)[0]
+        tm.assert_frame_equal(expected, res)
```

Comment:
Typically we use `result` as the first argument to this function and `expected` as the second. Would be nice to do that in these tests so failure output follows that pattern

Reply:
Okay, I've renamed variables (and fiddled with whitespace) in many, many tests.